numba/tests/test_range.py

```diff
@@ -177,6 +187,31 @@ def test_range_attrs(self):
         for arg in arglist:
             self.assertEqual(cfunc(*arg), pyfunc(*arg))
 
+    @tag('important')
+    def test_range_contains(self):
+        pyfunc = range_contains
+        arglist = [(0, 0, 1),
+                   (0, -1, 1),
+                   (-1, 1, 1),
+                   (-1, 4, 1),
+                   (-1, 4, 10),
+                   (5, -5, -2),]
+
+        vallist = [-10, -6, -5, -4, -2, -1, 0,
+                1, 2, 4, 5, 6, 10, -1.1, False, True,
+                1 + 0j, 1 + 1j, 1.1 + 0j, 1.0 + 1.1j]
+
+        cfunc = njit(pyfunc)
+        for arg in arglist:
+            for val in vallist:
+                self.assertEqual(cfunc(val, *arg), pyfunc(val, *arg))
+
+        cfunc_obj = jit(pyfunc, forceobj=True)
```

Comment:
Yeah, not sure if you really need `forceobj=True` here.

Reply:
If not, you can add the objects to the loops above.

Reply:
Attempting to pass a `dict` into the `nopython` jitted functions causes an error since Numba does not know how to type the `dict` in `nopython` mode. I put `forceobj`there just to make it explicit, and given future changes to numba:

http://numba.pydata.org/numba-doc/latest/reference/deprecation.html#deprecation-of-object-mode-fall-back-behaviour-when-using-jit

Reply:
sounds good.